tools/Dockerfile

```diff
@@ -47,24 +32,16 @@ RUN mkdir -p /usr/share/licenses/eksctl && \
 ARG EKSCTL_VERSION=0.112.0
 ARG EKSCTL_SOURCE_URL="https://github.com/weaveworks/eksctl/archive/refs/tags/v${EKSCTL_VERSION}.tar.gz"
 
-ARG GOARCH
 ARG EKSCTL_BINARY_URL="https://github.com/weaveworks/eksctl/releases/download/v${EKSCTL_VERSION}/eksctl_Linux_${GOARCH}.tar.gz"
 
-ARG EKSA_VERSION=0.12.1
-ARG EKSA_SOURCE_URL="https://github.com/aws/eks-anywhere/archive/refs/tags/v${EKSA_VERSION}.tar.gz"
-ARG EKSA_RELEASE_NUMBER=23
-# TODO EKS-A currently does not provide an ARM64 binary, add the hash for it when they do
-ARG EKSA_BINARY_URL="https://anywhere-assets.eks.amazonaws.com/releases/eks-a/${EKSA_RELEASE_NUMBER}/artifacts/eks-a/v${EKSA_VERSION}/${GOOS}/amd64/eksctl-anywhere-v${EKSA_VERSION}-${GOOS}-amd64.tar.gz"
-
 USER builder
 WORKDIR /home/builder/
-# eksctl
-RUN mkdir eksctl && curl -L ${EKSCTL_SOURCE_URL} \
-      -o eksctl_${EKSCTL_VERSION}.tar.gz && \
-    grep eksctl_${EKSCTL_VERSION}.tar.gz \
+RUN mkdir eksctl && curl -L "${EKSCTL_SOURCE_URL}" \
+      -o "eksctl_${EKSCTL_VERSION}.tar.gz" && \
+    grep "eksctl_${EKSCTL_VERSION}.tar.gz" \
```

Comment:
Looks like there might be a mix of tabs and spaces here. The indentation probably looks the same in the local editor, but the diff view shows them at different levels.

Not related to your change, but if you do an update it would be nice to fix that to be consistent.

Reply:
No tabs here! This is by design to signify the continuation of a command. It might be more obvious if it looked like this instead:
```
RUN mkdir eksctl && \
    curl -L "${EKSCTL_SOURCE_URL}" \
      -o "eksctl_${EKSCTL_VERSION}.tar.gz" && \
    grep "eksctl_${EKSCTL_VERSION}.tar.gz" \
...
```